dexpv2/segmentation.py

```diff
@@ -46,7 +46,10 @@ def reconstruction_by_dilation(
 
 
 def fancy_otsu_threshold(
-    image: ArrayLike, remove_hist_mode: bool = False, min_foreground: float = 0.0
+    image: ArrayLike,
+    remove_hist_mode: bool = False,
+    min_foreground: float = 0.0,
+    max_foreground: float = 0.0,
```

Comment:
```suggestion
    max_foreground: Optional[float] = None,
```
and add `from typing import Optional`
I think this would be more explicit about this being optional than `0.0`